docs/connecting-to-a-lit-network/migrating-to-datil.md

```diff
@@ -11,7 +11,7 @@ Depending on the existing Lit network you are using, you should migrate to its c
 
 | Currently Available | Requires Payment | Minimum Lit Package Version | Your Current Network | Network to Migrate to | Description                                                  |
 |---------------------|------------------|-----------------------------|----------------------|-----------------------|--------------------------------------------------------------|
-| ❌                   | ✅                | n/a                         | `habanero`           | `datil-prod`               | Decentralized mainnet designed for production use cases      |
+| ✅                   | ✅                | n/a                         | `habanero`           | `datil-prod`          | Decentralized mainnet designed for production use cases      |
```

Comment:
should we include minimal package version here? (once confirmed)

Reply:
Sure, put `6.4.0` for now